content/applications/inventory_and_mrp/manufacturing/basic_setup/bill_configuration.rst

```diff
@@ -2,98 +2,259 @@
+- :guilabel:`Flexible Consumption`: specifies if components used can deviate from the quantity
+  defined on the |BoM|. Choose :guilabel:`Blocked` if operators **must** adhere strictly to the
+  |BoM| quantity. otherwise, choose :guilabel:`Allowed` or :guilabel:`Allowed with Warning`.
+- :guilabel:`Routing`: select the preferred warehouse's manufacturing operation type for products
+  produced in multiple warehouses. If left blank, this warehouse's `Manufacturing` operation type is
+  used by default.
+- :guilabel:`Analytic Distribution`: select pre-created :doc:`analytic distribution models
+  <../../../finance/accounting/reporting/analytic_accounting>` from the list to automatically record
+  the cost of manufacturing products in the chosen journal.
+- :guilabel:`Manuf Lead Time`: define the number of days needed to complete a |MO| from the date of
+  confirmation.
+- :guilabel:`Days to prepare Manufacturing Order`: number of days needed to replenish components, or
+  manufacture sub-assemblies of the product.
+
+.. seealso::
+   - :doc:`Analytic distribution <../../../finance/accounting/reporting/analytic_accounting>`
+   - :doc:`Lead times
+     <../../inventory/shipping_receiving/advanced_operations_shipping/scheduled_dates>`
+
+.. image:: bill_configuration/misc-tab.png
    :align: center
-   :alt: Copy Existing Operations feature.
+   :alt: Show the *Miscellaneous* tab of the BoM.
+
+
+Add by-products to BoMs
+=======================
 
-Add by-products to a bill of materials (BoM)
-============================================
+A *by-product* is a residual product that is created during production in addition to the main
+product of a |BOM|. Unlike the primary product, there can be more than one by-product on a |BOM|.
+
+To add by-products to a |BOM|, first enable the :guilabel:`By-Products` feature in
+:menuselection:`Manufacturing app --> Configuration --> Settings`. In the :guilabel:`Operations`
+section, tick the checkbox for :guilabel:`By-Products` to enable the feature.
+
+.. image:: bill_configuration/by-products.png
+   :align: center
+   :alt: "By Products" feature in the settings page.
 
-A *By-Product* is a residual product that is created during production in addition to the main
-product of a :abbr:`BoM (Bill of Materials)`. Unlike the primary product, there can be more than
-one by-product on a :abbr:`BoM (Bill of Materials)`.
+Once the feature is enabled, add by-products to a |BOM| by clicking the :guilabel:`By-products` tab.
+Click :guilabel:`Add a line`, and fill in the :guilabel:`By-product`, :guilabel:`Quantity`, and
+:guilabel:`Unit of Measure`. Optionally, specify a :guilabel:`Produced in Operation` for the
+by-product.
 
-To add by-products to a :abbr:`BoM (Bill of Materials)`, first enable the :guilabel:`By-Products`
-feature in :menuselection:`Manufacturing --> Configuration --> Settings --> Operations`.
+.. example::
+   The by-product, `Mush`, is created in the `Grind grapes` operation when producing `Red Wine`.
```

Comment:
I love this example! 😄

Fun fact– the _mush_ is called "Lees" https://www.wineenthusiast.com/basics/lees-wine-meaning/

Reply:
oooh I'm becoming more *cultured* :wink: just by reading about this!